### PR TITLE
[EN/ZH] Tournament CMT 4K 2018

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+The **Chinese osu!mania 4K Tournament 2018** (***CMT 4K 2018***) is the third one-on-one, osu!mania 4K specified Chinese regional tournament, hosted by CMT team.
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration | 2018-09-10/2018-09-29 |
+| Qualification Stage | 2018-10-06/2018-10-07 |
+| Drawings | 2018-10-07 (20:00 UTC+8) |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Live Drawings
+8 -> +08

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+| Group Stage | 2018-10-13/2018-10-14 |
+| Round of 16 | 2018-10-20/2018-10-21 |
+| Round of 8 | 2018-10-27/2018-10-28 |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: isn't this called Quarterfinals?

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+All prizes are sponsored by **[Arkaaalin](https://osu.ppy.sh/users/7477382)**.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![](/wiki/shared/GCrown.png) | 8 months of osu!supporter tag, unique userpage badge |
+| ![](/wiki/shared/SCrown.png) | 4 months of osu!supporter tag |
+| ![](/wiki/shared/BCrown.png) | 2 months of osu!supporter tag |
+
+## Staff
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Organisation

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: The Chinese osu!mania 4K Tournament 2018 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+| Team | Staff |
+| --: | :-- |
+| Tournament managers | ![][CN] [hans1999](/users/6679329), ![][CN] [Flow Light](/users/6705256) |
+| Coordinator | ![][CN] [SSV Normandy](/users/6475977) |
+| Map selectors | ![][US] [qqqant](/users/8976576), ![][SG] [Polytetral](/users/8612061), ![][US] [Trainer-](/users/9331760), ![][SG] [Raveille](/users/1388767) |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Map Selectors

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+| Streamers and referees | ![][CN] [aiyulu](/users/189617), ![][CN] [Rasis](/users/1209363), ![][CN] [Flow Light](/users/6705256), ![][CN] [kj415j45](/users/9367540) |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Streamers and Referees

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+| Commentators | ![][CN] [hans1999](/users/6679329), ![][CN] [Flow Light](/users/6705256), ![][CN] [Rasis](/users/1209363), ![][AU] [williammak](/users/11083079), ![][CN] [crazybrother](/users/8045083) |
+| Wiki maintenance | ![][CN] [kj415j45](/users/9367540) |
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Wiki

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+
+## Links
+
+- [Forum topic](/community/forums/topics/800125)
+- [QQ group](https://jq.qq.com/?_wv=1027&k=5IH0Tkr)
+- [Challonge](https://challonge.com/CMT2018)
+- Livestream
+  - bilibili: [aiyulu](https://live.bilibili.com/10183) (Primary), [Flow Light](https://live.bilibili.com/73668) (Secondary), [Rasis](https://live.bilibili.com/5333) (Back up).
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Backup

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+2. All matches will be held on Saturday or Sunday, UTC+8.
+3. Matches in group stage may overlap.
+4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the actual stage. If player has conflict with the scheduling, Tournament Management will try to amend the schedule so that it fairly accommodates for all the participant. However, no promises can be guaranteed:
+     - If player is in any other time zone, please notify the Management in advance.
+     - Apart from time zone reasons, the absent player(s) will be considered as forfeited, regardless of whether they have sent messages to the Management.
+5. Request of rematch from forfeited player(s) will not be accepted.
+
+
+[CN]: /wiki/shared/flag/CN.gif "China"
+[US]: /wiki/shared/flag/US.gif "America"
```
Comment (Joehuu) [inline wiki/Tournaments/CMT_4K/2018/en.md]: `"America"` -> `"United States"`

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+[UK]: /wiki/shared/flag/GB.gif "Great Britain"
```
Comment (Joehuu) [inline wiki/Tournaments/CMT_4K/2018/en.md]: `"Great Britain"` -> `"United Kingdom"`

File: wiki/Tournaments/CMT_4K/2018/en.md [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+
+1. In principle, each stage will be held on a single weekend.
+2. All matches will be held on Saturday or Sunday, UTC+8.
+3. Matches in group stage may overlap.
+4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the actual stage. If player has conflict with the scheduling, Tournament Management will try to amend the schedule so that it fairly accommodates for all the participant. However, no promises can be guaranteed:
+     - If player is in any other time zone, please notify the Management in advance.
+     - Apart from time zone reasons, the absent player(s) will be considered as forfeited, regardless of whether they have sent messages to the Management.
+5. Request of rematch from forfeited player(s) will not be accepted.
+
+
```
Comment (Joehuu) [inline wiki/Tournaments/CMT_4K/2018/en.md]: Remove the extra line.

**Pull request merged**